### ISSUE TITLE
ChatDownload mode downloads as JSON although extension is .txt

### ISSUE THREAD

**Issue by NilEinne** [lay295/TwitchDownloader#397]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Describe your issue here

Using an extension .txt for the output file with the ChatDownload mode on with 1.50.5 gives me JSON files, exact same files as if I use .json. 1.40.9 gives me the expected plain text file. 

To keep things simple I've used this command line to test
`--mode ChatDownload --output "JustTesting1635834935.txt" --id 1635834935`

I've also tried replacing the extension with .TXT and .meow just to check, as the documentation suggestions anything which isn't JSON or HTML should be plain text but a JSON file is still outputted. And a different video ID. With 1.50.5, using .html does give a HTML file. (1.40.9 gives a plain text file, as expected since the format wasn't supported then and the extension is not .json)

**Comment by NilEinne** [comment]
I think I worked out what the problem is and how to fix it although as noted in my pull request it is untested and by someone with limited coding knowledge. From the code, I worked out and confirmed that with 1.50.5, if you do not have any extension you output plain text. It's only when you add an extension that it is always JSON unless HTML.

**Comment by NilEinne** [comment]
From looking at the code, I was initially confused why it outputted JSON as looking at TwitchDownloaderCLI/Program.cs and TwitchDownloaderCore/ChatDownloader.cs, it seemed to me that if the output format isn't specified, it would likely just fail. I then worked out it probably comes from TwitchDownloaderCore/Options/ChatDownloadOptions.cs. Just wondering if the default to JSON there is intentional. From the comment I modified in TwitchDownloaderCLI/Program.cs, I think it use to default to JSON if not .txt but now does almost the opposite. However, I have no idea if the JSON default is important for other things so it's better to keep it in the CLI only.
